pkg/messages/predefined.go

```diff
@@ -136,6 +136,7 @@ var (
 	ErrTerminateWorkflow             = APIError{"error terminating workflow %s", "ERR_TERMINATE_WORKFLOW", http.StatusInternalServerError, grpcCodes.Internal}
 	ErrMissingWorkflowEventName      = APIError{"missing workflow event name", "ERR_WORKFLOW_EVENT_NAME_MISSING", http.StatusBadRequest, grpcCodes.InvalidArgument}
 	ErrRaiseEventWorkflow            = APIError{"error raising event on workflow %s", "ERR_RAISE_EVENT_WORKFLOW", http.StatusInternalServerError, grpcCodes.Internal}
+	ErrPurgeWorkflow                 = APIError{"error purging workflow %s", "ERR_PURGE_WORKFLOW", http.StatusInternalServerError, grpcCodes.Internal}
```

Comment:
Surround the `%s` with quotes, like this:
```suggestion
	ErrPurgeWorkflow                 = APIError{"error purging workflow '%s'", "ERR_PURGE_WORKFLOW", http.StatusInternalServerError, grpcCodes.Internal}
```

I'm already making this change to all the other messages in another PR.